.gitmodules

```diff
@@ -14,3 +14,7 @@
 	path = imported/micro-manager
 	url = https://github.com/precice/micro-manager.git
 	branch = develop
+[submodule "imported/fmi-runner"]
+	path = imported/fmi-runner
+	url = https://github.com/precice/fmi-runner.git
+	branch = develop
```

Comment:
Thank you for the instructions. It mostly worked out. I merged the upstream PR to use the `develop` branch and used it here.

The report from `git submodule update --remote --merge` was a bit different:
```
Submodule path 'imported/fmi-runner': merged in '680ce077d4349c43d2181d9f34316887df71af7e'

```
But I see the `docs` folder in `imported/fmi-runner` and an entry was added to the `.gitmodules` file

Reply:
Now the CI seems to be fetching the submodule, but Jekyll still complains that `No subproject found at imported/fmi-runner/docs`.

The submodule (as I see in the diff view of this PR) seems to be pointing to https://github.com/precice/fmi-runner/tree/4c93437febe0399575396513efe0ea7bcdb66023

I think that, now that you merged the upstream, you need to update the submodule and it should work.

Reply:
Does the latest commit solve the problem?

Reply:
The module seems to be pointing to the latest commit, so everything should be fine.

But the CI has not been executed on the last commit, so I can only guess that.

Note that there are some merge conflicts you could resolve. This would also trigger the CI.

Reply:
Thank you, I missed that.

Reply:
The CI succeeds now! Should I merge, or wait for anything else?

Reply:
Please merge, thanks!